spot-client/src/common/remote-control/xmpp-connection.js

```diff
@@ -544,8 +570,7 @@ export default class XmppConnection extends Emitter {
 
         if ((error === 'connection.droppedError'
             || error === 'connection.otherError'
-            || error === 'item-not-found'
-            || error === 'conflict')
+            || error === 'item-not-found')
```

Comment:
This is an interesting case. I wonder in what practical cases conflict can be triggered here.

Reply:
Right now only ever I saw the 'conflict' triggered is when trying to join MUC with the same nickname.

In general it's safer to handle less cases here than more, because it is less risk of ending up in a broken loop.

If you mean when conflict can happen in real life then the cases I've found are:
- the case when another Spot TV is already in the MUC
- the case when network goes down, there's only ghost connection left. This one looks dangerous, but when network goes down the error will be different so the xmpp connection will go into the retry loop and when it hits the conflict on join MUC it will keep trying until after the idle timeout

I haven't found any other cases. It would have to be some other "conflict" returned not on the join MUC level and in such case it's better to break the loop for safety.